run/endpoints-v2-backend/src/test/resources/application.yml

```diff
@@ -0,0 +1,15 @@
+#  Copyright 2020 Google LLC

```

Comment:
This file shouldn't be needed, since found `run/endpoints-v2-backend/src/main/resources/application.yml `

Reply:
okay, i will delete application.yml - that means theres no properties file

Reply:
unless by "this file" you mean `application.properties`?

![image](https://user-images.githubusercontent.com/60039494/97600842-e11b5b80-19df-11eb-93ad-a5866aff24e2.png)

Reply:
You have this both in `/src/main/resources` and `/src/test/resources` only the file in `/src/main/resources` is needed